fili-sql/src/main/java/com/yahoo/bard/webservice/sql/presto/DruidQueryToPrestoConverter.java

```diff
@@ -162,17 +162,22 @@ protected SqlTimeConverter buildSqlTimeConverter() {
                     LOG.debug(msg);
                     return new RuntimeException(msg);
                 }))
-                .map(sqlAggregation -> builder.aggregateCall(
-                        sqlAggregation.getSqlAggFunction(),
-                        false,
-                        null,
-                        // TODO: investigate the field name case discrepency
-                        CaseFormat.LOWER_UNDERSCORE.to(
-                                CaseFormat.LOWER_CAMEL,
-                                sqlAggregation.getSqlAggregationAsName()
-                        ),
-                        builder.field(sqlAggregation.getSqlAggregationFieldName())
-                ))
+                .map(sqlAggregation -> {
+                    if (sqlAggregation.getSqlAggFunction() == SqlStdOperatorTable.COUNT) {
+                        return builder.countStar("count");
```

Comment:
I don't think we want to hardcode the name here, I think we should be able to take it from the aggregation
```suggestion
                        return builder.countStar(sqlAggregation.getSqlAggregationAsName());
```

Reply:
Also should probably add a test around this